core/scripts/sql/update/user.sql

```diff
@@ -1,3 +1,4 @@
 USE `texera_db`;
 ALTER TABLE user MODIFY name varchar(32) AFTER uid;
-ALTER TABLE user ADD email varchar(256) AFTER name;
\ No newline at end of file
+#ALTER TABLE user ADD email varchar(256) AFTER name;
```

Comment:
Why comment it out? Please be more informative here so that one can know which line to use to update his current database.

Reply:
Updated. Only the last line is needed in production.